bigquery/tests/unit/test_client.py

```diff
@@ -5161,3 +5161,166 @@ def test__do_multipart_upload_wrong_size(self):
+
+        with open_patch as _mock_file:
+            actual = client.schema_from_json(mock_file_path)
+            _mock_file.assert_called_once_with(mock_file_path)
+            # This assert is to make sure __exit__ is called in the context
+            # manager that opens the file in the function
+            _mock_file().__exit__.assert_called_once_with(None, None, None)
+
+        assert expected == actual
+
+    def test_schema_from_json_with_file_object(self):
+        from google.cloud.bigquery.schema import SchemaField
+
+        file_content = """[
+          {
+            "description": "quarter",
+            "mode": "REQUIRED",
+            "name": "qtr",
+            "type": "STRING"
+          },
+          {
+            "description": "sales representative",
+            "mode": "NULLABLE",
+            "name": "rep",
+            "type": "STRING"
+          },
+          {
+            "description": "total sales",
+            "mode": "NULLABLE",
+            "name": "sales",
+            "type": "FLOAT"
+          }
+        ]"""
+
+        expected = list()
+        expected.append(SchemaField("qtr", "STRING", "REQUIRED", "quarter"))
+        expected.append(
+            SchemaField("rep", "STRING", "NULLABLE", "sales representative")
+        )
+        expected.append(SchemaField("sales", "FLOAT", "NULLABLE", "total sales"))
+
+        client = self._make_client()
+
+        fake_file = io.StringIO(file_content)
+        actual = client.schema_from_json(fake_file)
+
+        assert expected == actual
+
+    def test_schema_to_json_with_file_path(self):
+        from google.cloud.bigquery.schema import SchemaField
+
+        file_content = """[
+  {
+    "description": "quarter",
+    "mode": "REQUIRED",
+    "name": "qtr",
+    "type": "STRING"
+  },
+  {
+    "description": "sales representative",
+    "mode": "NULLABLE",
+    "name": "rep",
+    "type": "STRING"
+  },
+  {
+    "description": "total sales",
+    "mode": "NULLABLE",
+    "name": "sales",
+    "type": "FLOAT"
+  }
+]"""
+        schema_list = list()
+        schema_list.append(SchemaField("qtr", "STRING", "REQUIRED", "quarter"))
+        schema_list.append(
+            SchemaField("rep", "STRING", "NULLABLE", "sales representative")
+        )
+        schema_list.append(SchemaField("sales", "FLOAT", "NULLABLE", "total sales"))
+
+        client = self._make_client()
+        mock_file_path = "/mocked/file.json"
+
+        open_patch = mock.patch("builtins.open", mock.mock_open())
+        with open_patch as _mock_file:
+            actual = client.schema_to_json(schema_list, mock_file_path)
+            _mock_file.assert_called_once_with(mock_file_path, mode="w")
+            _mock_file().write.assert_called_once_with(file_content)
+            # This assert is to make sure __exit__ is called in the context
+            # manager that opens the file in the function
+            _mock_file().__exit__.assert_called_once_with(None, None, None)
```

Comment:
We don't care about the actual arguments, so just `assert_called_once` will work.

Reply:
Removed the arguments from the assert.